README.md

```diff
@@ -33,6 +33,7 @@ PLUGINS_CONFIG = {
         "raw_notification_initial_days_since": 100,
         "raw_notification_size": 16384,
         "dashboard_n_days": 30,  # Defaults to 30 days in the configurations, change/override here
+        "overlap_job_log_only_impacts": False, # Defaults for False to alert on any notification, regardless of impact. True if there needs to be a known impact.
```

Comment:
Sorry @jvanderaa for the confusion, I think I didn't explain me well.
Impacts are always known, as being a Pydantic class (in the parser), it always comes from a good place.
My point was to "alarm" or not, depending on the type of the IMPACT, leaving the `NO_IMPACT` circuits out of the check, because no issue is expected on them.

Reply:
What do we want to put here for a name. We are good, but I think we can update the var name appropriately.